### DISCUSSION TITLE
Show feature sequences

### DISCUSSION THREAD

**Discussion by marta-coronado** [GMOD/jbrowse-components#2514] [Q&A]
Hello! I'm starting with JBrowse2. I've encountered a couple of scenarios that I can't figure it out how to address them. Sorry if my questions are dumb, but I wasn't able to solve them with the guide and I'm still learning :).

I've added a track displaying transposable elements annotations from a gff3 file. Is there a way to display the sequence of the transposable element under the feature details panel? (similar to transcript features) JBrowse1 was able to display the "sequence region" but I can't figure out how to do it with JBrowse2.

Also, I'd like that when I zoom out and display a large region of a chromosome, to display a feature density histogram for genes/transposable elements annotations, instead of showing the "force load" message.

Any help is much appreciated. Thank you very much!

**Comment by cmdcolin** [comment]
@marta-coronado great question

currently we don't display the sequence of arbitrary elements in the details but we could (we made a special feature sequence detail for genes/transcripts, but didn't make it generic yet)

>Also, I'd like that when I zoom out and display a large region of a chromosome, to display a feature density histogram for genes/transposable elements annotations, instead of showing the "force load" message.

good to know...currently no way to do this besides manually creating a bigwig of your file's feature densities and using it separately from your track, but we may want to come up with some idea for this built in!

**Reply by marta-coronado** [reply]
Thank you very much for your time Colin. Hope to see it in a future update!
